src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/SALauncher.java

```diff
@@ -301,33 +302,40 @@ private static void runJSTACK(String[] oldArgs) {
     }
 
     private static void runJMAP(String[] oldArgs) {
-        Map<String, String> longOptsMap = Map.of("exe=", "exe",
-                                                 "core=", "core",
-                                                 "pid=", "pid",
-                                                 "connect=", "connect",
-                                                 "heap", "-heap",
-                                                 "binaryheap", "binaryheap",
-                                                 "dumpfile=", "dumpfile",
-                                                 "histo", "-histo",
-                                                 "clstats", "-clstats",
-                                                 "finalizerinfo", "-finalizerinfo");
+        Map<String, String> longOptsMap = Map.ofEntries(
+           Map.entry("exe=", "exe"),
+           Map.entry("core=", "core"),
+           Map.entry("pid=", "pid"),
+           Map.entry("connect=", "connect"),
+           Map.entry("heap", "-heap"),
+           Map.entry("binaryheap", "binaryheap"),
+           Map.entry("dumpfile=", "dumpfile"),
+           Map.entry("gz=", "gz"),
+           Map.entry("histo", "-histo"),
+           Map.entry("clstats", "-clstats"),
+           Map.entry("finalizerinfo", "-finalizerinfo"));
```

Comment:
Indentation seems to be 3 instead of 4. And I think it's actually suppose to be 8 in this case.